src/views/Profile/components/AchievementRow/index.tsx

```diff
@@ -42,25 +42,21 @@ const AchievementRow: React.FC<AchievementRowProps> = ({ achievement, onCollectS
   const [isCollecting, setIsCollecting] = useState(false)
   const TranslateString = useI18n()
   const pointCenterContract = usePointCenterIfoContract()
-  const { account } = useWallet()
+  const { account } = useWeb3React()
   const { toastError, toastSuccess } = useToast()
 
-  const handleCollectPoints = () => {
-    pointCenterContract.methods
-      .getPoints(achievement.address)
-      .send({ from: account })
-      .on('sending', () => {
-        setIsCollecting(true)
-      })
-      .on('receipt', () => {
-        setIsCollecting(false)
-        onCollectSuccess(achievement)
-        toastSuccess('Points Collected!')
-      })
-      .on('error', (error) => {
-        toastError('Error', error?.message)
-        setIsCollecting(false)
-      })
+  const handleCollectPoints = async () => {
+    const tx = pointCenterContract.getPoints(achievement.address, { from: account })
```

Comment:
For calls like this, is it required to send `{ from: account }` ?

Reply:
Yeah, the original codebase with web3.js was look like this
pointCenterContract.methods
      .getPoints(achievement.address)
      .send({ from: account })

It's required to send ({from: account})